task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +104,18 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+        while (true) {
```

Comment:
do not use while(true) if you don't need infinite loop. Replace it with while(exit-condition) or for this case since you have fixed number of iterations it's better to use for-loop

Reply:
Thank you. I replaced it to for-loop.